integrations/svelte.md

```diff
@@ -29,6 +29,28 @@ Add `"svelte.plugin.css.diagnostics.enable": false` to your VS Code configuratio
 
 Add <kbd>[svelte-windicss-preprocess]</kbd> to your bundler configuration.
 
+### Svelte Kit (As of 1.0.0-next.100)
+You should only need to install the vite plugin, not the svelte-preprocess plugin - (`npm i -D vite-plugin-windicss`). Then add the plugin to your svelte config. Don't forget to add `import "virtual:windi.css"` to the top of your $layout.svelte!
```

Comment:
Change to something like, you also should include an very basic example of  `$layout.svelte` file

Svelte Kit uses vite as a bundler, therefore we suggest to use our [vite plugin](https://windicss.org/integrations/vite.html)
Add the plugin to your svelte config. 
After that add `import "virtual:windi.css"` to the top of your $layout.svelte file.